util/include/json_output.hpp

```diff
@@ -22,6 +22,8 @@
 
 namespace concordUtils {
 
+// Note: these templated functions are deprecated.  Use the BuildJson class below for new code.
```

Comment:
nit: if they deprecated why are you leaving them? Havn't you replaced them everywhere?

Reply:
@glevkovich No, there are many other places where the templated functions are called.  I fixed the places where nested JSON objects were created, which were incorrect.  In the other places where they are called they produce correct JSON.

I agree that leaving the old ones in place is ugly. I considered replacing them all, but that turns out to be a pretty big job and I was concerned that doing that might introduce bugs.  I asked Gadi about the wisdom of changing everything at once, and he advised not doing it.  I filed a "technical debt" ticket to remove all of them "later".